metadata.json

```diff
@@ -1,87 +1,67 @@
 {
-  "name": "google-glogging",
-  "version": "0.1.3",
-  "author": "Google",
-  "summary": "A Puppet module to manage Google Stackdriver Logging Agent",
+  "name": "ops-cloud_ops",
+  "version": "0.1.0",
+  "author": "Google Inc.",
+  "summary": "Manage Google Cloud Ops Agent",
   "license": "Apache-2.0",
-  "source": "https://github.com/GoogleCloudPlatform/puppet-google-logging",
-  "project_page": "https://github.com/GoogleCloudPlatform/puppet-google-logging",
-  "issues_url": "https://github.com/GoogleCloudPlatform/puppet-google-logging/issues",
+  "source": "https://github.com/BlueMedora/google-puppet-agents",
```

Comment:
`https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-puppet` (We plan to rename this repo to `google-cloud-ops-agents-puppet`)?